lib/v8.js

```diff
@@ -87,7 +87,7 @@ function getHeapSpaceStatistics() {
   const buffer = heapSpaceStatisticsBuffer;
   updateHeapSpaceStatisticsArrayBuffer();
 
-  for (var i = 0; i < kNumberOfHeapSpaces; i++) {
+  for (let i = 0; i < kNumberOfHeapSpaces; i++) {
```

Comment:
There are a number of performance related reasons why we haven't migrated fully over to `let` within loops. We would need to benchmark these kinds of changes.